starboard/android/shared/launcher.py

```diff
@@ -288,6 +288,9 @@ def Run(self):
     # Clear logcat
     self._CheckCallAdb('logcat', '-c')
 
+    # Remove the old package
+    self._CheckCallAdb('uninstall', _APP_PACKAGE_NAME)
```

Comment:
Does the check call succeed if there is nothing to uninstall?

Reply:
This will cause failure, updated the code.